### DISCUSSION TITLE
afterpay container in product page with breeze theme

### DISCUSSION THREAD

**Discussion by hardikkukadiya** [breezefront/community#254] [1. Help] [answered]
how to add afterpay container in product page with breeze theme?

**Comment by vovayatsyuk** [comment]
What is afterpay container?

Did you mean [afterpay module](https://github.com/afterpay/afterpay-magento-2)?

**Reply by hardikkukadiya** [reply]
Yes https://github.com/afterpay/afterpay-magento-2
Not working in product page, minicart and cart page after adding breeze theme and module.

**Comment by vovayatsyuk** [comment]
Can you please contact [swissuplabs.com support](https://swissuplabs.com/contacts/) and provide the MerchantID and MerchantKey settings so we can work on integration?

**Comment by vovayatsyuk** [comment, marked as answer]
Our integration module is ready: https://github.com/breezefront/module-breeze-afterpay
